eventhub/Module.md

```diff
@@ -58,33 +25,44 @@ event hubs. It includes operations to revoke publisher, get revoked publishers,
 
 * An Event Hub https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-create#create-an-event-hub 
 
+### Obtaining Tokens
 * Connection String of the Event Hub Namespace
```

Comment:
Extracting management credentials

Reply:
is this admin user? or is it called management credentials?

Reply:
We extract Shared Access Signature (SAS) authentication credentials. I changed the sentences to give the proper meaning in df7c75b . Thank you.